src/renderer/js/component/shapeShift/internal/active-shift.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { PureComponent } from "react";
+import { shell, clipboard } from "electron";
+import QRCode from "qrcode-react";
+import * as statuses from "constants/shape_shift";
+import { Address } from "component/common";
+import Link from "component/link";
+
+export default class ActiveShapeShift extends PureComponent {
+  constructor() {
+    super();
+    this.continousFetch = undefined;
+  }
+
+  componentDidMount() {
+    const { getActiveShift, shiftDepositAddress } = this.props;
+
+    getActiveShift(shiftDepositAddress);
+    this.continousFetch = setInterval(() => {
+      getActiveShift(shiftDepositAddress);
+    }, 10000);
+  }
+
+  componentWillUpdate(nextProps) {
+    const { shiftState } = nextProps;
+    if (shiftState === statuses.COMPLETE) {
+      this.clearContinuousFetch();
+    }
+  }
+
+  componentWillUnmount() {
+    this.clearContinuousFetch();
+  }
+
+  clearContinuousFetch() {
+    clearInterval(this.continousFetch);
+    this.continousFetch = null;
+  }
+
+  render() {
+    const {
+      shiftCoinType,
+      shiftDepositAddress,
+      shiftReturnAddress,
+      shiftOrderId,
+      shiftState,
+      shiftDepositLimit,
+      clearShapeShift,
+      doShowSnackBar,
+    } = this.props;
+
+    return (
+      <div className="shapeshift__active-shift">
+        {shiftState === statuses.NO_DEPOSITS && (
+          <div>
+            <p>
+              Send up to{" "}
+              <span className="font-bold">
```

Comment:
Can the existing `credit-amount` classes be used for this? I'm generally opposed to this type of classname-as-styling, though you can certainly find other examples in the code. If `credit-amount` does not work, then I would instead propose new CSS classes specifically for styling financial values, not `font-bold`.